src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -503,7 +512,7 @@ private void AddInfo(string message, Action<IOperationProgress> onProgress)
                     token: TaskCancelToken.Token);
 
                 smuggler.Execute();
-                file.Flush(flushToDisk: true);
+                stream.Flush();
```

Comment:
We really care about the `FlushToDIsk`, mind, so we need to call the outer stream anyway